stable/superset/values.yaml

```diff
@@ -13,8 +13,18 @@ image:
   pullSecrets: []
 
 initFile: |-
-  /usr/local/bin/superset-init --username admin --firstname admin --lastname user --email admin@fab.org --password admin
-  superset run
+  if [ "$1" == "development-mode" ]; then
+    /usr/local/bin/superset-init --username admin --firstname admin --lastname user --email admin@fab.org --password admin
+    superset run
+  elif [ "$1" == "production-mode" ]; then
+    gunicorn -b 0.0.0.0:8088 \
+      --limit-request-line 0 \
+      --limit-request-field_size 0 \
+      --timeout 300 superset:app
+  else
+    echo "Need to specify which mode to start in"
```

Comment:
in the echo, you should include the options `development-mode || production-mode`

Reply:
Include the command for starting in dev mode or prod mode, as in how would a developer read this ... and possibly copy and paste the first time (tbh). This isn't a nit either in my view. It's just about reducing the surface area of complexity in the chart so developers can get up and running with no snags, especially the ones that already use the chart.

Reply:
I have updated documentation and error message to be a bit clearer.
I'm not sure I understand everything you ask for in the last comment though. Copy and paste the first time?

No I get it, I also want charts to have high quality so your comments are appreciated!

By the way, what happens to this chart when this repo is decommissioned?

Reply:
Apache Superset repo. I contribute there as well, quite a bit.  This chart will live for a while.

Reply:
Thanks a lot for the contribution!